spec/features/step_definitions/web_steps.rb

```diff
@@ -0,0 +1,219 @@
+# Taken from the cucumber-rails project.
```

Comment:
you aren't using cucumber, don't include files like this that are not necessary (they do more harm than good) - note also that your directory structure here is strange.  The contents of step_definitions and support directories are completely redundant and should be removed.